resources/setup-files-automation/README.md

```diff
@@ -37,13 +37,26 @@ If the repo contains all the required files you need, to start:
 
 To navigate to directory with the script run:
 
-   `cd /resources/testing `
+   `cd /resources/setup-files-automation `
 
 To run the script,execute the command : 
 
    `./automation-script.sh`
 
-You will be shown an output in your terminal similar to below:
+This automation script will run an initial setup script that will ask:
+
+     `Do you want to update your email address? (y/n):`
+
+     If the answer is y, it will prompt you with
```

Comment:
This line isn't part of your script output, so I'd get this outside of the 'monospace' text and as normal text.

Reply:
Updated, kind eagle eyed fellow